packages/@aws-cdk/aws-autoscaling/lib/auto-scaling-group.ts

```diff
@@ -864,3 +889,143 @@ export interface MetricTargetTrackingProps extends BaseTargetTrackingProps {
    */
   readonly targetValue: number;
 }
+
+export interface BlockDevice {
+  /**
+   * The device name exposed to the EC2 instance
+   *
+   * @example '/dev/sdh', 'xvdh'
+   *
+   * @see https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/device_naming.html
+   */
+  readonly deviceName: string;
+
+  /**
+   *
+   */
+  readonly volume: BlockDeviceVolume;
+
+  /**
+   * Suppresses the device mapping.
+   * If this property is set to true for the root device,  the instance might fail the Amazon EC2 health check.
+   * Amazon EC2 Auto Scaling launches a replacement instance if the instance fails the health check.
+   *
+   * @default device mapping is left untouched
+   */
+  readonly noDevice?: boolean;
```

Comment:
Can we avoid the double-negative. Would `mappingEnabled` work (with a `true` default)

Reply:
Done